receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -1 +1,272 @@
+    enum:
+    - read
+    - write
+  gc_type:
+    description: Type of garbage collection
+    enum:
+    - young
+    - old
+  memory_type:
+    description: Type of memory
+    enum:
+    - heap
+    - non-heap
+  direction:
+    description: Data direction
+    enum:
+    - received
+    - sent
+  document_type:
+    description: Type of document
+    enum:
+    - active
+    - deleted
+  shard_type:
+    description: State of the shard
+    enum:
+    - active
+    - relocating
+    - initializing
+    - unassigned
+  operation_type:
+    description: Type of operation
+    enum:
+    - index
+    - delete
+    - get
+    - query
+    - fetch
+    - scroll
+    - suggest
+    - merge
+    - refresh
+    - flush
+    - warmer
+  thread_pool_name:
+    description: Thread pool name
+  thread_state:
+    description: State of the thread
+    enum:
+    - active
+    - idle
+  task_state:
+    description: State of the task_state
+    enum:
+    - rejected
+    - completed
+metrics:
+  # these metrics are from /_nodes/stats, and are node level metrics
+  elasticsearch.node.cache.memory.usage:
+    description: Size in bytes of the cache.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name]
+  elasticsearch.node.cache.evictions:
+    description: The number of evictions from the cache.
+    unit: "{evictions}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name]
+  elasticsearch.node.jvm.gc.collections.count:
+    description: The number of GC collections performed by the JVM.
+    unit: "{collections}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [gc_type]
+  elasticsearch.node.jvm.gc.collections.time:
+    description: Total time spent by the JVM running the GC.
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [gc_type]
+  elasticsearch.node.jvm.memory.usage:
+    description: Memory used by the JVM.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [memory_type]
+  elasticsearch.node.jvm.threads.count:
+    description: Number of open threads in the node's JVM process.
+    unit: "{threads}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.jvm.threads.peak:
+    description: Maximum number of concurrently running threads in the node's JVM process.
+    unit: "{threads}"
+    gauge:
+      value_type: int
+    attributes: []
+  elasticsearch.node.fs.disk.usage:
+    description: The amount of used disk space across all file stores for this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.fs.disk.free:
```

Comment:
This should likely be combined.

Reply:
I folded `elasticsearch.node.fs.disk.free` into `elasticsearch.node.fs.disk.usage`, since that seems to be the suggestion in the guidelines https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-naming